### PR TITLE
Add 'Building Source' to docs

### PR DESCRIPTION
**Pull request by DeveloperWOW64**
## Description

This PR adds a new item to the documentation. 'Building Files from source'. It tells users how to build Files from the repository.

## Motivation and Context

This helps users learn to build Files easily so people don't need to keep asking.

## How Has This Been Tested?

It has been tested by running GH pages on my repo. (https://developerwow64.github.io/files-community.github.io/)

### CODE REVIEW

File: docs/articles/building-source.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Building Files from source
+
+1. Install Microsoft Visual Studio and the UWP Development kit.
+2. Clone the [repository](https://github.com/files-community/files) and open `Files.sln` in VS
+3. Make sure that `Files.Package` is set as the startup item
+4. Run `Files.Package`, x64, Release and it should look like this:
```
Comment (yaichenbaum) [inline docs/articles/building-source.md:6]:
```suggestion
4. Run `Files.Package`, x64, Debug and it should look like this:
```

**Review by yaichenbaum — CHANGES_REQUESTED** [review]

File: docs/articles/building-source.md (hunk 1 above)
Comment (DeveloperWOW64) [reply docs/articles/building-source.md:6]: For me debug doesn't work

**Comment by DeveloperWOW64** [comment]
Updated to 'Debug' I updated the image as well.

File: docs/articles/building-source.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Building Files from source
+
+1. Install Microsoft Visual Studio and the UWP Development kit.
+2. Clone the [repository](https://github.com/files-community/files) and open `Files.sln` in VS
+3. Make sure that `Files.Package` is set as the startup item
+4. Run `Files.Package, x64` as `Debug` and it should look like this:
+![VS Run config image](https://raw.githubusercontent.com/DeveloperWOW64/files-community.github.io/main/img/vs_run_cfg.png)
```
Comment (yaichenbaum) [inline docs/articles/building-source.md:7]: Can you include this image in the PR?

**Review by yaichenbaum — CHANGES_REQUESTED** [review]

File: docs/articles/building-source.md (hunk 2 above)
Comment (DeveloperWOW64) [reply docs/articles/building-source.md:7]: wdym?

Comment (chingucoding) [reply docs/articles/building-source.md:7]:
```suggestion
![VS Run config image](../../img/vs_run_cfg.png)
```

I think he means this.

File: docs/articles/building-source.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Building Files from source
+
+1. Install Microsoft Visual Studio and the UWP Development kit.
+2. Clone the [repository](https://github.com/files-community/files) and open `Files.sln` in VS
+3. Make sure that `Files.Package` is set as the startup item
```
Comment (yaichenbaum) [inline docs/articles/building-source.md:5]: Can you provide instructions on how someone can set the correct startup project?

Comment (DeveloperWOW64) [reply docs/articles/building-source.md:5]: what do you mean?

Comment (yaichenbaum) [reply docs/articles/building-source.md:5]: Step 3 is to set Files.Package as the startup item, can you provide instructions on how to do that?

Comment (DeveloperWOW64) [reply docs/articles/building-source.md]: Done!

**Review by yaichenbaum — APPROVED** [review]

File: docs/articles/building-source.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# Building Files from source
+
+1. Install Microsoft Visual Studio and the UWP Development kit.
```
Comment (chingucoding) [inline docs/articles/building-source.md:3]: Any version of Visual Studio?

File: docs/articles/building-source.md [hunk 5]
```diff
@@ -0,0 +1,8 @@
...
+2. Clone the [repository](https://github.com/files-community/files) and open `Files.sln` in VS.
+3. Make sure that `Files.Package` is set as the startup item by right-clicking on `Files.Package` in the solution explorer and hitting 'Set as Startup item'.
+4. Run `Files.Package, x64` as `Debug` and it should look like this:
```
Comment (chingucoding) [inline docs/articles/building-source.md:6]: Should be generic as in "Choose the architecture for your system"

File: docs/articles/building-source.md (hunk 4 above)
Comment (DeveloperWOW64) [reply docs/articles/building-source.md:3]: Wdym?

Comment (chingucoding) [reply docs/articles/building-source.md:3]: Should it be 2019 and above? Or is VS 6 fine too (I doubt that but you get the idea)

File: docs/articles/building-source.md (hunk 5 above)
Comment (DeveloperWOW64) [reply docs/articles/building-source.md]: How about now?
`Install Microsoft Visual Studio and the UWP Development kit.
Clone the repository and open Files.sln in VS.
Make sure that Files.Package is set as the startup item by right-clicking on Files.Package in the solution explorer and hitting 'Set as Startup item'.
Run Files.Package, x64* as Debug and it should look like this: VS Run config image
You're good to go!
If x64 doesn't work, use the architecture of your system`

Comment (DeveloperWOW64) [reply docs/articles/building-source.md]: How about now?
image

File: docs/articles/building-source.md (hunk 4 above)
Comment (DeveloperWOW64) [reply docs/articles/building-source.md:3]: yes, should be VS2019

Comment (DeveloperWOW64) [reply docs/articles/building-source.md:3]: and above

Comment (DeveloperWOW64) [reply docs/articles/building-source.md:3]: nooo, yair just said it was 17 and above
```suggestion
1. Install Microsoft Visual Studio 2017 or later and the UWP Development kit.
```

File: docs/articles/building-source.md [hunk 6]
```diff
@@ -0,0 +1,11 @@
+# Building Files from source
+
+1. Install Microsoft Visual Studio and the UWP Development kit.
+2. Clone the [repository](https://github.com/files-community/files) and open `Files.sln` in VS.
+3. Make sure that `Files.Package` is set as the startup item by right-clicking on `Files.Package` in the solution explorer and hitting 'Set as Startup item'.
+4. Run `Files.Package, x64`* as `Debug` and it should look like this:
```
[the remark this answers is not in the document]
Comment (DeveloperWOW64) [reply docs/articles/building-source.md:6]:
```suggestion
4. In the architecture pane, select `Files.Package, x64`* as `Debug` which should look like this:
```
I don't want it to be too vague so is this good?

**Review by chingucoding — APPROVED** [review]
Thank you for adding this @DeveloperWOW64 !!!

**Pull request merged**